### DISCUSSION TITLE
I real pro format import for chord charts / layouts

### DISCUSSION THREAD

**Discussion by MrShinyBoots** [ChurchApps/ChurchAppsSupport#328] [FreeShow] [answered]
I can easily add an image of an ireal pro chord chart however this has more steps to do in the back ground - image export, colour inversion - cropping, re saving the image etc. 

would it be possible to support ireal pro format import - or a music xml import for a chords widget ( grid based text box essentially) 
looking on github a few people have done various projects including an html5  ireal pro renderer 

https://github.com/daumling/ireal-renderer?tab=readme-ov-file

- this feature would give access to a lot of song charts as well as the ability to create lead sheets on ireal pro, publish them and then import them directly into freeshow - possibly with the ability to transpose them on the fly (or change to a more nashville numbers system / scale degree system) on the fly

edit: you could also implement a search function similar to the lyrics function based on how I find charts in Ireal pro currently (their) app uses a google search to their forums 

many thanks.

**Comment by vassbo** [comment]
Thanks. Would be possible, but not prioritized. :)

**Comment by vassbo** [comment, marked as answer]
Looking into this more, I don't think this will be done as FreeShow supports ChordPro format & text chord editing. And you can use the screen capture if you don't have the images ready to import! So unless someone opens a PR, or there are more requests this won't be done unfortunately!
